
  int32 upperleft
  int32 lowerRight
  string color
  string cameraID